### PR TITLE
App architecture post

### PR DESCRIPTION
**Pull request by ljhaywar**

### CODE REVIEW

File: source/article/active-active-application-architectures-with-mongodb.txt [hunk 1]
```diff
@@ -0,0 +1,592 @@
...
+.. ARTICLE CONTENT
+.. ---------------------------------------------------------------
+
+====================================================
+Active-Active Application Architectures with MongoDB
+====================================================
+
+.. content::
+
+    Introduction
```
Comment (schmalliso) [inline source/article/active-active-application-architectures-with-mongodb.txt]: We have an `.. introduction` directive that I'd suggest you use for the introduction (it won't change anything with the render, it'll just let us tag the content more correctly and then be able to improve delivery if the design changes.

File: source/article/active-active-application-architectures-with-mongodb.txt [hunk 2]
```diff
@@ -0,0 +1,592 @@
...
+    always see the previous write operation, regardless of which replica is
+    serving the request. By enforcing strict, causal ordering of operations
+    within a session, causal consistency ensures every read is always
+    logically consistent, enabling monotonic reads from a distributed system – guarantees that cannot be met by most multi-node databases. Causal
+    consistency allows developers to maintain the benefits of strict data
+    consistency enforced by legacy single node relational databases, while
+    modernizing their infrastructure to take advantage of the scalability
+    and availability benefits of modern distributed data platforms.
+
+    Conclusion
```
Comment (schmalliso) [inline source/article/active-active-application-architectures-with-mongodb.txt]: .. conclusion:: (same comment as the introduction one above)

**Comment by ljhaywar** [comment]
Thanks for reviewing!

I added the intro and summary directives in a new commit.

The images are rendering for me: http://docs-devhub-staging.s3-website-us-east-1.amazonaws.com/ec82c87/devhub/docsworker-xlarge/app-arch/article/active-active-application-architectures-with-mongodb.  No idea why they wouldn't be loading for you.

**Comment by schmalliso** [comment]
Rebased & merged!

**Pull request closed without merging**